app/views/pages/privacy.html.erb

```diff
@@ -51,31 +51,33 @@
 
     <h2>Services We Use</h2>
     <p>
-      Like many other websites, we sometimes use cookies, Google Analytics and Bugnag to help us make our websites better.
+      Like many other websites, we sometimes use cookies<% if Rails.application.secrets.google_analytics_id.present? %>, Google Analytics<% end %> and Bugnag to help us make our websites better.
     </p>
     <p>
       These tools are very common and widely used by other sites, but they do have privacy implications. When you visit sites that use these tools, you are potentially telling companies such as Google that you visited, as well as some information about how your web browser is configured. If you don’t want to share your browsing activities on sites with other companies, you can install opt-out browser plugins. You might find this article useful: <a href="http://www.guardian.co.uk/technology/askjack/2011/jun/17/ask-jack-internet-privacy-web-browsers-cookies">The Guardian – How to Keep Your Privacy Online</a>.
     </p>
 
     <h3>Cookies</h3>
     <p>
-      Cookies are small amounts of text stored on your computer's web browser that can be written by websites and read by websites. They can also be shared between websites. We use cookies to store some information about your current session, this enables us to do things like keep you logged into the website between pages. Cookies are also used by third-party providers like Google:
+      Cookies are small amounts of text stored on your computer's web browser that can be written by websites and read by websites. They can also be shared between websites. We use cookies to store some information about your current session, this enables us to do things like keep you logged into the website between pages. <% if Rails.application.secrets.google_analytics_id.present? %>Cookies are also used by third-party providers like Google:<% end %>
```

Comment:
might want to triple check no session info from cookies sent on to bugsnag. Might be safer to keep this clause in completely and not name a third party provider in the case that we add something in the future adn forget to change this (or add something like Danger/Peril)

Reply:
Updated to always say "third parties" instead of specifically (and conditionally) Google.